doc/lambdapi.bnf

```diff
@@ -33,6 +33,7 @@ QID ::= [UID "."]+ UID
             | "coerce_rule" <rule> ";"
             | "unif_rule" <unif_rule> ";"
             | "notation" <qid_or_nat> <notation> ";"
+            | "opaque" <qid_or_nat> ";"
```

Comment:
Just to be sure: did you generate this file with `make bnf`?

Reply:
Yes I did, I just did not push the new .bnf